<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excitation measurement</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="upperGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>141</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>140</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="startEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="stopEdit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="pwrChk">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>90</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Power</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="stepEdit">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="startButt">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="spcChk">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Spectrum</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="scanProgBar">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>201</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pwrTimeEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>110</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>110</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Power coll. time:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="showPwrmButt">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Powermeter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameEdit">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>70</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>scan.txt</string>
       </property>
      </widget>
      <widget class="QLabel" name="locLabel">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>37</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>./</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>100</y>
         <width>58</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="formatCombo">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>100</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>ASCII XY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ASCII X</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>float x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>double x</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="locButt">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>40</y>
         <width>88</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveButt">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>70</y>
         <width>88</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="showSpctrmButt">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>10</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Spectrometer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="showChiraButt">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>10</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Chirascan</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="autoFolderChk">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>100</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>AutoCreate folders</string>
       </property>       
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="PlotWidget" name="graphicsView"/>
    </item>
    <item>
     <widget class="QGroupBox" name="lowerGroup">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>131</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>530</width>
        <height>130</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="pwrRadioBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Power ref</string>
       </property>
       <widget class="QRadioButton" name="powerRefFile">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>From file</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="powerRefNone">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>None</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="powerRefCur">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Current</string>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="sxmaxEdit">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>50</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1023</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>169</x>
         <y>20</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sum xmin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>20</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>xmax:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sxminEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>50</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
